### PR TITLE
Add SysMonTask

### PR DESCRIPTION
**Pull request by ryanfortner**
SysMonTask is a Linux system monitor similar to the Windows task manager, written in Python.

https://github.com/KrispyCamel4u/SysMonTask

### CODE REVIEW

**Review by Itai-Nelken — APPROVED** [review]

**Comment by Botspot** [comment]
```
Package installation complete.
The folder you are executing pip from can no longer be found.
Failed to upgrade psutil
```

**Comment by Botspot** [comment]
```
Package installation complete.
The folder you are executing pip from can no longer be found.
Failed to upgrade psutil
```

**Comment by ryanfortner** [comment]
> ```
> Package installation complete.
> The folder you are executing pip from can no longer be found.
> Failed to upgrade psutil
> ```

Will Sysmontask still launch without running `pip3 install -U psutil`? I know that it won't launch without this step on my buster system but I haven't tried bullseye.

**Comment by Botspot** [comment]
> > ```
> > Package installation complete.
> > The folder you are executing pip from can no longer be found.
> > Failed to upgrade psutil
> > ```
> 
> Will Sysmontask still launch without running `pip3 install -U psutil`?

Yes it did launch. Strange.

**Comment by Itai-Nelken** [comment]
That's because psutil is probably already installed. Maybe from commanderpi which also uses it.

**Comment by Crilum** [comment]
Yeah, It works fine for me, even with the error.. I do have psutil installed, so that's why.. CPU temp doesn't show though..

File: apps/SysMonTask/install [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+install_packages https://apt.raspbian-addons.org/debian/pool/main/s/sysmontask/sysmontask_1.3.9-ubuntu20.10_all.deb || exit 1
+
+sudo pip3 install -U psutil || error "Failed to upgrade psutil"
```
Comment (Crilum) [inline apps/SysMonTask/install:5]:
```suggestion
pip3 install -U psutil || error "Failed to upgrade psutil"
```
Found the problem.. pip isn't supposed to be run with sudo.

Comment (Itai-Nelken) [reply apps/SysMonTask/install:5]: Oh, thats right! Running it as sudo installs it for the root user, not the current user.

Comment (Itai-Nelken) [reply apps/SysMonTask/install:5]: @ryanfortner

**Comment by Botspot** [comment]
We're getting many error reports that contain this:
```
The folder you are executing pip from can no longer be found.
Failed to upgrade psutil
```
Any ideas?

**Comment by Botspot** [comment]
I found the problem. It turns out the `install_packages` function was leaving the working directory as `/tmp/pi-apps-local-packages`!

**Pull request merged**